.storybook-s2/docs/Migrating.jsx

```diff
@@ -0,0 +1,770 @@
+          <div>{'\n'}</div>
+          <div className={style({backgroundColor: 'red-200'})}>{`- <ActionButton marginStart="size-100">`}</div>
+          <div className={style({backgroundColor: 'green-200'})}>{`+ <ActionButton styles={style({marginStart: 8})}>`}</div>
+          <div>{`    Edit`}</div>
+          <div>{`  </ActionButton>`}</div>
+        </pre>
+
+        <H3>Border width</H3>
+        <P>Affected style props: <Code>borderWidth</Code>, <Code>borderStartWidth</Code>, <Code>borderEndWidth</Code>, <Code>borderTopWidth</Code>, <Code>orderBottomWidth</Code>, <Code>borderXWidth</Code>, <Code>borderYWidth</Code>.</P>
+        <P>Example:</P>
+        <pre className={'sb-unstyled ' + style({padding: 32, marginY: 32, backgroundColor: 'layer-1', borderRadius: 'xl', fontFamily: 'code', fontSize: 'code-sm', lineHeight: 'code'})}>
+          <div className={style({backgroundColor: 'red-200'})}>{`- <View borderWidth="thin"  />`}</div>
+          <div className={style({backgroundColor: 'green-200'})}>{`+ <div className={style({borderWidth: 1})} />`}</div>
+        </pre>
+        <P>Border widths should be updated to use pixel values. Use the following mappings:</P>
+        <table>
+          <tr>
+            <th>Spectrum 1</th>
+            <th>Spectrum 2</th>
+          </tr>
+          <tr>
+            <td><Code>'none'</Code></td>
+            <td><Code>0</Code></td>
+          </tr>
+          <tr>
+            <td><Code>'thin'</Code></td>
+            <td><Code>1</Code></td>
+          </tr>
+          <tr>
+            <td><Code>'thick'</Code></td>
+            <td><Code>2</Code></td>
+          </tr>
+          <tr>
+            <td><Code>'thicker'</Code></td>
+            <td><Code>4</Code></td>
+          </tr>
+          <tr>
+            <td><Code>'thickist'</Code></td>
+            <td><Code>'[8px]'</Code></td>
+          </tr>
+        </table>
+
+        <H3>Border radius</H3>
+        <P>Affected style props: <Code>borderRadius</Code>, <Code>borderTopStartRadius</Code>, <Code>borderTopEndRadius</Code>, <Code>borderBottomStartRadius</Code>, <Code>borderBottomEndRadius</Code>.</P>
+        <P>Example:</P>
+        <pre className={'sb-unstyled ' + style({padding: 32, marginY: 32, backgroundColor: 'layer-1', borderRadius: 'xl', fontFamily: 'code', fontSize: 'code-sm', lineHeight: 'code'})}>
+          <div className={style({backgroundColor: 'red-200'})}>{`- <View borderRadius="small"  />`}</div>
+          <div className={style({backgroundColor: 'green-200'})}>{`+ <div className={style({borderRadius: 'sm'})} />`}</div>
+        </pre>
+        <P>Border radius values should be updated to use pixel values. Use the following mappings:</P>
+          <table>
+          <tr>
+            <th>Spectrum 1</th>
+            <th>Spectrum 2</th>
+          </tr>
+          <tr>
+            <td><Code>'size-0'</Code></td>
+            <td><Code>0</Code></td>
+          </tr>
```

Comment:
Was this a valid value? I just see the following: 
<img width="813" alt="image" src="https://github.com/user-attachments/assets/489d4c58-ef4b-4699-ae5b-a87416f4a7a2">